### PR TITLE
Adding a continue button to the telephone appointment form

### PR DESCRIPTION
**Pull request by Guntrisoft**
After a user selected a time we were moving them on automatically
to the customer form page.

Feedback from DWP content team suggested that the standard
gov.uk way is to let a user make a choice and then decide to
continue to the next step by clicking a 'continue' button

screen shot 2017-04-12 at 13 00 06

### CODE REVIEW

File: app/controllers/telephone_appointments_controller.rb [hunk 1]
```diff
@@ -34,7 +34,11 @@ def create_step_2
     if request.xhr?
       render partial: 'times', locals: { times: @times }
     else
-      telephone_appointment.advance! { render :new }
+      if telephone_appointment.start_at.nil?
```
Comment (benlovell) [inline app/controllers/telephone_appointments_controller.rb]: Can we flip this logic around and say `if telephone_appointment.start_at?` instead?

File: app/views/telephone_appointments/new.html.erb [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
           Unable to submit the form
         </h2>
+
         <p>Check the following:</p>
 
         <ul class="error-summary-list">
           <% @telephone_appointment.errors.each do |key, message| %>
             <li><%= link_to "#{TelephoneAppointment.human_attribute_name key} – #{message}", "#telephone_appointment_#{key}" %></li>
           <% end %>
+          <% if @telephone_appointment.start_at.nil? %>
```
Comment (benlovell) [inline app/views/telephone_appointments/new.html.erb]: I think it's clearer to switch this around and state `unless @telephone_appointment.start_at?` here.

File: app/controllers/telephone_appointments_controller.rb (hunk 1 above)
Comment (Guntrisoft) [inline app/controllers/telephone_appointments_controller.rb]: Done

File: app/views/telephone_appointments/new.html.erb (hunk 2 above)
Comment (Guntrisoft) [inline app/views/telephone_appointments/new.html.erb]: Done

**Comment by Guntrisoft** [comment]
[Add green continue button under times on 'Book a phone appointment' page](https://trello.com/c/4i3IfOQ3/100-add-green-continue-button-under-times-on-book-a-phone-appointment-page)

**Comment by Guntrisoft** [comment]
@benlovell added `slot_selected?` method

**Pull request merged**